runtime/near-vm-runner-standalone/src/main.rs

```diff
@@ -67,6 +68,27 @@ struct StandaloneOutput {
     pub state: State,
 }
 
+fn default_vm_context() -> VMContext {
+    return VMContext {
+        current_account_id: "alice".to_string(),
+        signer_account_id: "bob".to_string(),
+        signer_account_pk: vec![0, 1, 2],
+        predecessor_account_id: "carol".to_string(),
+        input: vec![],
+        block_index: 0,
+        block_timestamp: 0,
```

Comment:
This is just some current time in nanoseconds
```suggestion
        block_timestamp: 1586796191203000000,
```